### ISSUE TITLE
Item class

### ISSUE THREAD

**Issue by naor2013** [Amulet-Team/Amulet-Core#33]
An Item class should be created for use in Entities that have an item parameter. It should include the needed Item properties in it. It can be a data class.

**Comment by StealthyExpertX** [comment]
I also have an idea for a few small methods for the item class,
is_enchanted(item) will take an item object and check if the item is enchanted and return true if the 
item is not enchanted will return false.

Something simular to this would work.
`def is_enchanted(item):
	if "tag" in item:
		if "ench" in item["tag"]:
			return True
		return False
	return False`

**Comment by Podshot** [comment]
I think that's a good idea! I'll work on that once #36 is completed `impl-entities` branch is merged into master

**Comment by gentlegiantJGC** [comment]
I have been putting it off but I think it would make sense to do versioning for this in the translation library. As far as I am aware items are all blocks with some additions/modifications

**Comment by gentlegiantJGC** [comment]
After a bit more thought I think items should be left as NBT and the translation library should deal with them how it wants internally but should be given and return NBT.
I am going to close this issue because I don't think Amulet should have a specific class representation of an Item. This is partially due to it being stored within an NBT object and only NBT objects are allowed to be stored within other NBT objects
